.github/workflows/triage_handle_new_comments.yml

```diff
@@ -0,0 +1,33 @@
+name: 'Handle Issue/PR Comment Workflow'
+on:
+  # makes this workflow reusable
+  workflow_call:
+    secrets:
+      ADD_TO_TRIAGE_BOARD_TOKEN:
+        required: true
+
+  issue_comment:
+    types: [created]
+
+jobs:
+  handle-comment-scenarios:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          repository: 'cypress-io/release-automations'
+          ref: 'master'
+          ssh-key: ${{ secrets.WORKFLOW_DEPLOY_KEY }} 
+      - name: Set up Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: '16'
+      - name: Run comment_workflow.js Script
+        uses: actions/github-script@v6
+        with:
+          github-token: ${{ secrets.ADD_TO_TRIAGE_BOARD_TOKEN }}
+          script: |
+            const script = require('./scripts/triage/comment_workflow.js')
+            script.handleComment(github, context);
+          
```

Comment:
```suggestion
```

Reply:
ha, @mschile always wants one line at the end of files for easier merging.

Reply:
I like an end of line, just the extra tabs 😋 usually the linter yells at you which is why I noticed. We must not lint these files